content/posts/2023/reana-0.9.1.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "REANA 0.9.1 is released"
+date: 2023-09-28T09:00:00+01:00
+---
+
+We are happy to announce REANA 0.9.1, a minor update that allows users to preview PDF and ROOT files from the web interface, makes it easier to clean up old workflows to free disk space, adds support for integrating with Keycloak, and more!
+
+<!--more-->
+
+## What's new for the users?
+
+### PDF/ROOT preview
+
+REANA 0.9.1 improves the preview of workspace files from the web interface by adding support for PDF and ROOT files.
+You will be able to interact with ROOT plots directly from your browser, without the need to download the files on your computer.
+
+![ROOT file preview](/images/reana-0.9.1-root-preview.png)
+
+### Pruning workspaces
+
+REANA already provides tools to keep disk usage under control, for example by setting up automatic retention rules or by deleting old workflows.
+REANA 0.9.1 also introduces the new `prune` command, which can be used to clean up your workflows by deleting all the files in their workspaces, with the exclusion of input and output files, so that you don't have to delete each file one by one.
+
+```console
+$ reana-client prune -w reana-demo-root6-roofit
+==> SUCCESS: The workspace has been correctly pruned.
+```
+
+Take a look to the related [Prune the workspace](https://docs.reana.io/advanced-usage/user-quotas/#prune-the-workspace) documentation page to learn more about this new command.
+
+### ... and much more!
+
+This new release brings many more small improvements. For example, it's now possible to stop workflows from the web UI, by clicking on the _Stop workflow_ button in the actions menu.
+Furthermore, restarted Snakemake workflows now report the correct total number of jobs, by excluding cached jobs that were simply reused from previous runs in the workspace and not really executed.
+
+## What's new for the administrators?
+
+### Keycloak
+
+REANA provides users with two ways to log in, either using username/password or via an external SSO provider.
+Up until now support for external SSO systems was limited to CERN SSO, but REANA 0.9.1 removes this limitation by being able to integrate with any custom Keycloak instance.
+
+If you are interested in using Keycloak together with REANA, please take a look to the related [Keycloak Single Sign-On configuration](https://docs.reana.io/administration/configuration/configuring-access/#keycloak-single-sign-on-configuration) documentation page.
+
+### Launcher examples
+
+REANA 0.9.0 introduced the possibility to launch workflows from external sources like GitHub, GitLab or Zenodo.
+It is now possible to customise the demo examples that are showcased in the Launch-on-REANA welcome page via the new `components.reana_ui.launcher_examples` Helm value.
+In this way you can expose your own demos to your users, instead of the default ones provided by REANA.
+
+![Launcher examples](/images/reana-0.9.1-launcher-examples.png)
```

Comment:
Perhaps the browser picture would profit from borders?  Otherwise it looks "too whitish" when compared to the other picture above.  Perhaps use the simulated browser window borders here?